Python/Product/PythonTools/Properties/AssemblyInfo.cs

```diff
@@ -29,20 +29,9 @@
 [assembly: CLSCompliant(false)]
 [assembly: NeutralResourcesLanguage("en-US")]
 
-[assembly: ProvideCodeBase(AssemblyName = "Microsoft.PythonTools", CodeBase = "Microsoft.PythonTools.dll", Version = AssemblyVersionInfo.StableVersion)]
```

Comment:
They're all in Core. We need an assembly reference to generate it, but PythonTools cannot reference assemblies that reference PythonTools because it's circular.